docs/api/app.md

```diff
@@ -108,8 +108,20 @@ Returns:
 * `event` Event
 * `hasVisibleWindows` Boolean
 
-Emitted when the application is activated, which usually happens when clicks on
-the applications's dock icon.
+Emitted when the application is activated, which usually happens when the user clicks on
+the application's dock icon.
+
+### Event: 'continue-activity' _OS X_
+
+Returns:
+
+* `event` Event
+* `type` String - A string identifying the event. Maps to [`NSUserActivity.activityType`](https://developer.apple.com/library/ios/documentation/Foundation/Reference/NSUserActivity_Class/index.html#//apple_ref/occ/instp/NSUserActivity/activityType).
+* `userInfo` Object - Contains app-specific state stored by the activity on another device.
```

Comment:
I mean, I actually think it's quite reasonable to constrain it to strings, it'd probably make your code way simpler